### ISSUE TITLE
Option to set the language icon depending of the project type

### ISSUE THREAD

**Issue by BluTree** [Discord-RP/Visual-Studio#3]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
An option to set the language icon depending of the project. For example, set the C# icon when a csproj is opened, c++ for vcxproj, etc...

**Comment by VisualDevelopment** [comment]
Is there a list of these extensions? Would be helpful.

**Comment by BluTree** [comment]
So, I looked for infos on google, and it appears that the project type is defined by a GUID in its file, rather than an extension.
I found the function `DTE.ActiveSolutionsProjects` that get an array of all the projects in the solution opened. And then there is a  `Project.Kind` that returns this GUID. 
I also found a website that references nearly all the GUID of vs : https://www.codeproject.com/Reference/720512/List-of-Visual-Studio-Project-Type-GUIDs
